### PR TITLE
Added laravel 5 compatible service provider

### PR DESCRIPTION
**Pull request by ajaaibu**
Created a new service provider to add support for Laravel 5.

### CODE REVIEW

File: src/League/ColorExtractor/Laravel/ColorExtractorServiceProviderL5.php [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    protected $defer = false;
+
+    /**
+     * Bootstrap the application events.
+     *
+     * @return void
+     */
+    public function boot()
+    {
+ //       $this->package('league/color-extractor');
```
Comment (toin0u) [inline src/League/ColorExtractor/Laravel/ColorExtractorServiceProviderL5.php]: It's better to remove the line than to comment it :)

File: src/League/ColorExtractor/Laravel/ColorExtractorServiceProviderL5.php [hunk 2]
```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace League\ColorExtractor\Laravel;
+
+use Illuminate\Support\ServiceProvider;
+use League\ColorExtractor\Client;
+
+/**
+ * @author Antoine Corcy <contact@sbin.dk>
```
Comment (toin0u) [inline src/League/ColorExtractor/Laravel/ColorExtractorServiceProviderL5.php]: Maybe you're name is more appropriate here :)

**Comment by toin0u** [comment]
Maybe some unit tests can be nice to have as well :)

**Comment by zandzpider** [comment]
Would be lovely if this could get merged or looked at. thanks

File: src/League/ColorExtractor/Laravel/ColorExtractorServiceProviderL5.php (hunk 1 above)
Comment (GrahamCampbell) [inline src/League/ColorExtractor/Laravel/ColorExtractorServiceProviderL5.php]: Just remove the entire boot function.

Comment (GrahamCampbell) [inline src/League/ColorExtractor/Laravel/ColorExtractorServiceProviderL5.php]: This can also be removed from the L4 service provider anyway.

**Comment by GrahamCampbell** [comment]
This service provider isn't even needed. The original one does just fine. See https://github.com/thephpleague/color-extractor/pull/30.

**Comment by toin0u** [comment]
:+1:

**Comment by GrahamCampbell** [comment]
Ping @MatTheCat regarding replacing this PR with #30.

**Pull request closed without merging**